docs/markdown/snippets/devenv.md

```diff
@@ -0,0 +1,11 @@
+## Dump devenv into file and select format
+
+`meson devenv --dump [<filename>]` command now takes an option filename argument
```

Comment:
"now takes an option filename" makes it sound like -- dump always existed and the filename was added.